lms/djangoapps/edxnotes/api_urls.py

```diff
@@ -5,7 +5,7 @@
 
 from django.conf.urls import url
 
-from edxnotes import views
+from lms.djangoapps.edxnotes import views
```

Comment:
I didn't bother to sort these because I figure we'll do this in one big `isort` sweep after the sys path hacks are truly gone.